Core/tests/Unit/Exception/ServiceExceptionTest.php

```diff
@@ -41,4 +42,194 @@ public function testDoesntHaveServiceException()
         $ex = new ServiceException('foo');
         $this->assertFalse($ex->hasServiceException());
     }
+
+    public function testGetErrorInfoMetadataWithGrpc()
+    {
+        if (!extension_loaded('grpc')) {
+            $this->markTestSkipped('Must have the grpc extension installed to run this test.');
+        }
+
+        $metadata = [
+            'ackId1' => 'PERMANENT_FAILURE_INVALID_ACK_ID',
+            'ackId2' => 'PERMANENT_FAILURE_INVALID_ACK_ID'
+        ];
+
+        $errorInfoRow = [
+            "@type" => "google.rpc.errorinfo-bin",
```

Comment:
```suggestion
            '@type' => 'google.rpc.errorinfo-bin',
```

Could we aim for consistency between `'` and `"`?